index.html

```diff
@@ -166,11 +169,21 @@ <h2 class="about_primary_text">About me</h2>
             </div>
             <div class="about_bottom">
                 <div class="text_box">I'm always interested in hearing about new projects, so if you'd like to chat please get in touch.</div>
-                <form action="" class="about_form">
-                    <input type="text" placeholder="Full name">
-                    <input type="text" placeholder="Email address">
-                    <textarea name="" id="" cols="30" rows="10">Hey I’d like to connect to...</textarea>
-                    <button class="get_in_touch"><a href="#">Get in touch</a></button>
+                <form action="https://formspree.io/f/xeqwroeb" class="about_form" method="post">
+                    <ul>
+                        <li>
+                            <input type="text" name="user_name" id="name" placeholder="Full name" maxlength="30" required>
+                        </li>
+                        <li>
+                            <input type="email" name="email" id="email" placeholder="Email address" required>
+                        </li>
+                        <li>
+                            <textarea name="textarea" id="textarea" maxlength="500" required>Hey I’d like to connect to...</textarea>
+                        </li>
+                        <li>
+                            <button type="submit" class="get_in_touch">Get in touch</button>
+                        </li>
```

Comment:
- [ ] You did a great job 😃 at adding the contact form, but kindly as stated by the previous reviewer please try to add the hover effect to the submit button this will make it nice when the user hovers it, so please 😊 add the hover effect to make it like the Figma Design 💯👌

| effects| 
| --- |
|    ![image](https://user-images.githubusercontent.com/83360636/228486085-24ba18cf-a5b9-46d0-8d04-11e1257c9d08.png)   |